src/lib/viewers/controls/settings/SettingsCheckboxItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash/uniqueId';
+import './SettingsCheckboxItem.scss';
+
+export type Props = {
+    className?: string;
+    isChecked: boolean;
+    label: string;
+    onChange: (isChecked: boolean) => void;
+};
+
+export default function SettingsCheckboxItem({ className, isChecked, label, onChange }: Props): JSX.Element {
+    const { current: id } = React.useRef(uniqueId('bp-SettingsCheckboxItem_'));
+
+    const handleChange = (event: Event<HTMLInputElement>): void => {
+        onChange(event.target.checked);
+    };
+
+    return (
+        <div className={classNames('bp-SettingsCheckboxItem', className)}>
+            <input
+                checked={isChecked}
+                className="bp-SettingsCheckboxItem-input"
+                id={id}
+                onChange={handleChange}
+                type="checkbox"
+            />
+            <label className="bp-SettingsCheckboxItem-label" htmlFor={id}>
```

Comment:
Can we wrap the `input` with the `label` to avoid needing a unique id?

Reply:
I had attempted to do that but the linter still complained about the label needing to be associated with a control